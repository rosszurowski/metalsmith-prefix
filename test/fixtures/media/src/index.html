<!DOCTYPE html>
<html lang="en">
<head>
	<link href="styles.css">
	<link href="/more/styles.css">
</head>
<body>
	<img src="hello.jpg" />
	<img src="hello/world.jpg" />
	<img src="/hello/world.jpg" />
	<video>
		<source src="/source.mp4" />
	</video>
	<script src="jquery.min.js"></script>
	<script src="/bundle.js"></script>
</body>
</html>