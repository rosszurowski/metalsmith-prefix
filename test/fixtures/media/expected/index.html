<!DOCTYPE html>
<html lang="en">
<head>
	<link href="styles.css">
	<link href="/prefix/more/styles.css">
</head>
<body>
	<img src="hello.jpg">
	<img src="hello/world.jpg">
	<img src="/prefix/hello/world.jpg">
	<video>
		<source src="/prefix/source.mp4">
	</video>
	<script src="jquery.min.js"></script>
	<script src="/prefix/bundle.js"></script>
</body>
</html>